src/ad-sizes.ts

```diff
@@ -33,32 +67,30 @@ type SizeKeys =
 	| 'portrait'
 	| 'skyscraper';
 
-interface SizeMapping {
-	mobile?: AdSize[];
-	desktop?: AdSize[];
-	phablet?: AdSize[];
-	tablet?: AdSize[];
-}
+type SlotName =
+	| 'right'
+	| 'comments'
+	| 'inline'
+	| 'top-above-nav'
+	| 'mostpop'
+	| 'merchandising-high'
+	| 'merchandising-high-lucky'
+	| 'high-merch-paid'
+	| 'merchandising'
+	| 'survey'
+	| 'im'
+	| 'carrot'
+	| 'epic'
+	| 'mobile-sticky';
 
-interface SizeMappings {
-	right: SizeMapping;
-	comments: SizeMapping;
-	'top-above-nav': SizeMapping;
-	mostpop: SizeMapping;
-	'merchandising-high': SizeMapping;
-	merchandising: SizeMapping;
-	survey: SizeMapping;
-}
+type PlatformSize = 'mobile' | 'desktop' | 'phablet' | 'tablet';
```

Comment:
Should we rename this to `Breakpoint`? It would match the definitions in [Source](https://github.com/guardian/source/blob/main/packages/%40guardian/source-foundations/src/breakpoints/breakpoints.ts), which we may want to import and use in the future.